Frameworks/CoreGraphics/CGPattern.mm

```diff
@@ -198,3 +209,24 @@ CGPatternRef CGPatternRetain(CGPatternRef pattern) {
     CFRetain((id)pattern);
     return pattern;
 }
+
+CGRect _CGPaternGetBounds(CGPatternRef pattern) {
+    RETURN_RESULT_IF_NULL(pattern, CGRectNull);
+    return ((CGPattern*)pattern)->bounds;
+}
+
+void _CGPaternIssueCallBack(CGContextRef context, CGPatternRef pattern) {
+    FAIL_FAST_IF_NULL(context);
```

Comment:
no one should be calling this with null context, secondly I would like to know ASAP, given this is the meat of the logic.
I guess I can make it return and do nothing, and it will cascade into a null at some point. Earlier the better for this.

---
In reply to: [93077913](https://github.com/Microsoft/WinObjC/pull/1586#discussion_r93077913) [](ancestors = 93077913)